web/shapeworks/src/types/index.ts

```diff
@@ -33,8 +33,7 @@ export interface AnalysisPCA {
 
 export interface AnalysisParams {
     range: number,
-    step: number,
-    numSteps: number,
```

Comment:
Why remove step size? This is a necessary argument for the analysis command as I understand it